### ISSUE TITLE
Facebook source: Rendering Issue for FB Button Templates

### ISSUE THREAD

**Issue by lizhutter** [airyhq/airy#2116]
Currently, the message response is only seen in the overview, and not in the message view.

This is the payload:
```
{
      "id": "2f155a01-060b-5226-b2e2-2086724e8e98",
      "content": {
        "sender": {
          "id": "447960795603212"
        },
        "recipient": {
          "id": "3876242712503913"
        },
        "timestamp": 1624623317469,
        "message": {
          "mid": "m_mZ-SOME_ID",
          "is_echo": true,
          "text": "Hey FIRST_NAME, somtext.\n\nsomemoretext.",
          "metadata": "airy_bot",
          "app_id": 936557276482485,
          "attachments": [
            {
              "type": "template",
              "url": null,
              "title": "Hey FIRST_NAME, somtext.\n\nsomemoretext",
              "payload": {
                "template_type": "button",
                "buttons": [
                  {
                    "type": "web_url",
                    "url": "https://link.airy.co/c/8k1-Hkgvd",
                    "title": "Zum Filialfinder"
                  },
                  {
                    "type": "web_url",
                    "url": "https://link.airy.co/c/QZ-cb94-N",
                    "title": "zum Online Shop"
                  }
                ]
              }
            }
          ]
        }
      },
      "sent_at": "2021-06-25T12:15:17.469Z",
      "delivery_state": "delivered",
      "source": "facebook",
      "metadata": {},
      "from_contact": false
    },
```

We receive this payload from the Webhook as an echo, but it renders only as this in the Inbox:
<img width="341" alt="bildschirmfoto_2021-07-05_um_13 54 07-1" src="https://user-images.githubusercontent.com/74183162/124597941-13643f00-de32-11eb-893a-be8ce0d0ee3c.png">

So, the issue happens for those we get via the Webhook as an echo. This can be reproduced for all button templates via Facebook.

**Reminder**: the format for sending a button is different which is shown below:

```
{
    "attachment":{
      "type":"template",
      "payload":{
        "template_type":"button",
        "text":"What do you want to do next?",
        "buttons":[
          {
            "type":"web_url",
            "url":"https://www.messenger.com",
            "title":"Visit Messenger"
          },
          {
            ...
          },
          {...}
        ]
      }
    }
  }
```

Here we have `attachment.payload.text` but we receive it back as an echo as `message.text` and `message.attachments[0].title`

**Note**: we should not break the one for sending button templates when we fix it, just add the rendering for echoes.

**Comment by AudreyKj** [comment]
we've already fixed this in PR  #1995

**Comment by lizhutter** [comment]
okay great! thanks @AudreyKj
